README.md

```diff
@@ -43,11 +43,11 @@ Apollo is loaded with new modules and features but needs to be calibrated and co
 
 * Ubuntu 18.04
 
-* NVIDIA driver version 440.33.01 and above ([Web link](https://www.nvidia.com/Download/index.aspx?lang=en-us)), and CUDA version 10.2
+* NVIDIA driver version 440.33.01 and above ([Web link](https://www.nvidia.com/Download/index.aspx?lang=en-us))
 
 * Docker-CE version 19.03 and above ([Official doc](https://docs.docker.com/engine/install/ubuntu/))
 
-* NVIDIA container toolkit ([Official doc](https://github.com/NVIDIA/nvidia-docker))
+* NVIDIA Container Toolkit ([Official doc](https://github.com/NVIDIA/nvidia-docker))
 
```

Comment:
Where should we mention Apollo Software Installation Guide, here or the Installation section below ?

Reply:
We don't need to introduce the software installation guide in prerequisites.